phpBB/phpbb/db/extractor/base_extractor.php

```diff
@@ -22,9 +22,9 @@
 abstract class base_extractor implements extractor_interface
 {
 	/**
-	 * @var    \phpbb\filesystem\filesystem
+	 * @var    \phpbb\filesystem\temp
 	 */
-	protected $filesystem;
+	protected $temp;
```

Comment:
Call variables for this `$temp_dir` to have it more descriptive

Reply:
Is temp service, not temp dir

Reply:
True, I just don't like having such a short name that can mean more than just "temporary directory service"